test/cbmc/proofs/DNS/DNSgetHostByName_cancel/DNSgetHostByName_cancel_harness.c

```diff
@@ -51,21 +55,31 @@ void func( const char * pcHostName,
 
```

Comment:
Should func above be checking for null pointers, etc?

Reply:
Added.